pyramid_heroku/tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+from .test_herokuapp_access import TestHerokuappAccessTween
```

Comment:
Please use absolute, not relateive, imports. Easier to sort, and to move around, less problems with circular dependencies, easier for IDEs to parse, etc.